src/pages/Explorer/Components/EditDialog.svelte

```diff
@@ -32,9 +33,12 @@
         updateExplorerItem(item, title, description, isPublic)
         closeDialog()
       })
-      .catch((err) => {
-        // TODO: handle error and remove log
-        console.log(err.message)
+      .catch(() => {
+        notifications$.show({
+          variant: 'error',
+          title: 'Something went wrong',
+          description: 'Please try again or contact support',
+        })
```

Comment:
This error dispatcher is duplicated in multiple places (`src/pages/Explorer/Category/ExplorerCategory.svelte`, `src/pages/Explorer/Category/Widget.svelte`, etc). Move it to the function and reuse it